### PR TITLE
Add accessibility tests for individual enrollment steps

### PR DESCRIPTION
**Pull request by PaulMorris**
Add accessibility tests for the pages of the enrollment process for individuals. Includes some refactoring and an easy fix for one failing test (a color contrast issue).  

The changes in this PR are on top of those in the current version of PR #673.  The new commits start with "Move basic step definitions out of 'DataCoverageStepDefinitions'".

Tested by running the tests.  Those that are not `@ignore`d pass successfully.

Issue #518 Implement accessibility checking in CI

### CODE REVIEW

**Comment by cecilia-donnelly** [comment]
Note that this shouldn't be reviewed until #673 is merged and this is rebased on master.

**Comment by PaulMorris** [comment]
Rebased on master now that #673 has landed, so ready for review now.

**Comment by cecilia-donnelly** [comment]
Three of these fail for me locally, all for the same error classes: a few for "this form has no submit button," a few telling me that the dropdowns have insufficient contrast, and one for an href with no content (an enrollment with no NPI).  I don't know why I'd be seeing the insufficient contrast problem on dropdown menus if you aren't, @PaulMorris.  I know the empty href element is something we've seen before -- how did you fix that elsewhere?

- General Accessibility Checks. Advanced Search Page
- General Accessibility Checks. Dashboard Draft Page 
- General Accessibility Checks. Dashboard Page

**Comment by cecilia-donnelly** [comment]
I did get one failure here, on the dashboard page, for the same reasons given in #676.

**Comment by PaulMorris** [comment]
This revision adds a commit "Ignore some a11y tests that sometimes fail" which omits the a11y tests that were failing for @cecilia-donnelly -- most are dashboard pages due to issue #672 (empty href) and also the advanced search page (with "no submit button for form").  

Curiously, these tests were not added in this PR but in previous PRs #651 and #671.  It makes sense that the database-state-dependent ones (#672) might not have shown up before, but I don't know why the advanced search page issue is only coming up now.  (I haven't been able to reproduce it.)

File: psm-app/integration-tests/src/test/resources/features/general/accessibility.feature [hunk 1]
```diff
@@ -52,6 +62,8 @@ Feature: General Accessibility Checks
     Given I am on the Account Setup page
     Then I should have no accessibility issues
 
+  # can fail with 'no submit button for form'
```
Comment (kfogel) [inline psm-app/integration-tests/src/test/resources/features/general/accessibility.feature]: Is there any more detailed explanation available for this one, or more context to link to in an issue or pull request commentary or something?

Comment (PaulMorris) [reply psm-app/integration-tests/src/test/resources/features/general/accessibility.feature]: @kfogel No more details on this one yet.  I haven't been able to reproduce it locally.  I'll add an issue for it, and in that a link to the comment in the PR where @cecilia-donnelly mentions it.

Comment (kfogel) [reply psm-app/integration-tests/src/test/resources/features/general/accessibility.feature]: Perfect -- thanks @PaulMorris.

**Comment by PaulMorris** [comment]
Added issue number #687 to the comment about the ignored 'Advanced Search' page test, in commit  ac41eecd50bec158508fcbd202554ad0223792fa.

**Pull request merged**